ast/location/location.go

```diff
@@ -36,12 +36,6 @@ func (loc *Location) Errorf(f string, a ...interface{}) error {
 	return errors.New(loc.Format(f, a...))
 }
 
-// Wrapf returns a new error value that wraps an existing error with a message formatted
-// to include the location info (e.g., line, column, filename, etc.)
-func (loc *Location) Wrapf(err error, f string, a ...interface{}) error {
```

Comment:
Could we do `return fmt.Errorf(f+": %w", append(a, err))` instead? 🤔 We wouldn't have to remove the exported method then, I believe...

Reply:
Sure, I wasn't sure if this was actually used or necessary, but I suppose it's better not to drop an exported method if you can avoid it.